src/v/kafka/server/handlers/list_offsets.cc

```diff
@@ -139,7 +144,9 @@ static ss::future<list_offset_partition_response> list_offsets_partition(
                 }
```

Comment:
L138/L143 - does this need a translation?

Reply:
good catch, will add a `timequery` method to partition_proxy.